.github/components/dictionary.txt

```diff
@@ -190,26 +197,31 @@ SCE
 ScPCA
 SCPCP
 scRNA
+scRNAseq
 scrublet
 SEACells
 SemVar
 seq
 SingleR
 snRNA
 socio
+spearman
```

Comment:
A proper noun, so does this belong here?

Reply:
Yes, it will still need to be in the dictionary, but it should be uppercase for sure.